changes/8847.refpix.rst

```diff
@@ -0,0 +1 @@
+Include resample, pixel_replace, extract_1d, and combine_1d into MOS background pipeline
```

Comment:
This change note is in the wrong place - it should be written to changes/8847.master_background.rst, not refpix